src/deploy/functions/release/index.ts

```diff
@@ -59,16 +59,16 @@ export async function release(
     }
   }
 
-  const functionExecutor: executor.QueueExecutor = new executor.QueueExecutor({
+  const throttlerOptions = {
     retries: 30,
     backoff: 20000,
     concurrency: 40,
-    maxBackoff: 40000,
-  });
+    maxBackoff: 100000,
```

Comment:
Just curious, where did 100k come from?

Reply:
100000 is still a pretty wishy-washy number, I just noticed that retries were failing pretty often on very large deployments so I gave the maxBackoff a boost to increase retry success rates